src/legacy/core_plugins/visualizations/public/np_ready/public/plugin.ts

```diff
@@ -138,6 +144,8 @@ export class VisualizationsPlugin
     setUiActions(uiActions);
     setTimeFilter(data.query.timefilter.timefilter);
     setAggs(data.search.aggs);
+    setOverlays(core.overlays);
+    setChrome(core.chrome);
```

Comment:
Just a note that once you do the new platform cutover, all of these setters will need to also be set in `src/legacy/ui/public/new_platform/new_platform.ts` (and probably the corresponding karma mocks too)